src/types/Identification.ts

```diff
@@ -0,0 +1,11 @@
+import { Enumeration } from "./index";
+
```

Comment:
Should we consider renaming to PartyIdentification? 

We will have identifications for Products, Orders, Party etc.

Reply:
Yes Sir. We have the same pattern in Moqui as well
https://demo.moqui.org/qapps/tools/Entity/DataEdit/EntityList?filterRegexp=Identification&viewOption=all&submitButton=submitButton